git/hooks/pre-commit

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+files=`git diff --cached --name-status`
+re="gen-PKGBUILD.py"
+if [[ $files =~ $re ]]
+then
+  echo "Re-generating PKGBUILD..."
+  python gen-PKGBUILD.py > PKGBUILD
+  git add gen-PKGBUILD.py
+  echo "Re-generating .SRCINFO..."
+  makepkg --printsrcinfo > .SRCINFO
+  git add .SRCINFO
+  git commit --amend -C HEAD --no-verify
```

Comment:
You should verify this.  I had a failure before I removed the amend.  No commit had been made, and it was trying to amend the previous one, and failed to lock the index.